charts/chainlink-cluster/dashboard/cmd/deploy.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"github.com/K-Phoen/grabana/dashboard"
+	lib "github.com/smartcontractkit/chainlink/dashboard-lib/lib"
+	core_don "github.com/smartcontractkit/chainlink/dashboard-lib/lib/core-don"
+	k8spods "github.com/smartcontractkit/chainlink/dashboard-lib/lib/k8s-pods"
+	waspdb "github.com/smartcontractkit/wasp/dashboard"
+)
+
+const (
+	DashboardName = "Chainlink Cluster (DON)"
```

Comment:
Do we want to hardcore dashboard name ?

Reply:
I think we should have some vars static, there is no need to have more than one dashboard for `core` right now so I'd like to be sure no one deploys another dashboard @Atrax1 😄